android/src/test/java/com/reactnativecommunity/asyncstorage/next/ArgumentHelpersTest.kt

```diff
@@ -1,79 +0,0 @@
-package com.reactnativecommunity.asyncstorage.next
```

Comment:
why have you removed those tests?

Reply:
These are only testing `createNativeCallArguments`, and `createNativeCallArguments` is a test-only utility. They're not testing real code as far as I can see. Did I miss anything?

Reply:
Might seem like it, but these test are actually checking [.toEntryList](https://github.com/react-native-async-storage/async-storage/blob/master/android/src/main/java/com/reactnativecommunity/asyncstorage/next/ArgumentHelpers.kt#L8) and  [.toKeyList](https://github.com/react-native-async-storage/async-storage/blob/master/android/src/main/java/com/reactnativecommunity/asyncstorage/next/ArgumentHelpers.kt#L33) extension functions

Reply:
Aha! Let me restore them then.